config/puma.rb

```diff
@@ -0,0 +1,9 @@
+workers Integer(ENV['WEB_CONCURRENCY'] || 2)
+threads_count = Integer(ENV['RAILS_MAX_THREADS'] || 5)
```

Comment:
I think that the `Integer` constructor  was present in the default puma configuration because it will throw an ArgumentError exception if the argument isn't a valid integer.
The `to_i` method on the other hand try to convert it into an integer anyway.

Anyway the `to_i` choise for me is ok! :)